pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2760,6 +2760,7 @@ dont-distribute-packages:
   - boolector
   - ccelerate-cuda
   - cplex-hs
+  - cuda # 2020-08-18 because of dependency nvidia-x11
   - cublas
   - cufft
   - cusolver
```

Comment:
I think it makes sense to move it here, especially because things like cublas and cufft are here.

Although it is unfortunate that Hydra (or whatever is responsible for this), isn't smart enough to automatically figure out this shouldn't be built because it has dependencies that aren't free.